travis.sh

```diff
@@ -35,7 +35,8 @@ else
   echo 'External PR: skipping integration tests.'
 fi
 ./java-repo-tools/scripts/test-localhost.sh jetty helloworld-jsp
-./java-repo-tools/scripts/test-localhost.sh jetty helloworld-servlet
+# ./java-repo-tools/scripts/test-localhost.sh jetty helloworld-servlet
```

Comment:
I guess we will need to update the script to work with the new plugins. Is that one of the changes you were talking about?